README.md

```diff
@@ -412,6 +412,13 @@ These are the following services and checks perform by each Auditor. There are c
 | Amazon_AppStream_Auditor.py            | AppStream 2.0 (Users)          | Are users reported as Compromised                                                   |
 | Amazon_AppStream_Auditor.py            | AppStream 2.0 (Users)          | Do users use SAML authentication                                                    |
 | Amazon_CloudFront_Auditor.py           | CloudFront Distribution        | Does distribution have trusted signers with key pairs                               |
+| Amazon_CloudFront_Auditor.py           | CloudFront Distribution        | Does distribution have Origin Shield enabled                                        |
```

Comment:
Just needs an update to the total count at the top of table, think we should be at 318 in total with the new CF checks

Reply:
@jonrau1 just made an update to the README. Let me know if that doesn't work. Sorry for such a long delay.